packages/database/src/migrations/20210412210749-AddLesmisEntityVitalsReports-modifies-data.js

```diff
@@ -0,0 +1,216 @@
+'use strict';
+
+import { codeToId, insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+const SCHOOL_REPORT_CODE = 'LESMIS_school_vitals';
+const MULTI_SCHOOL_REPORT_CODE = 'LESMIS_multi_school_vitals';
+const DISTRICT_REPORT_CODE = 'LESMIS_sub_district_vitals';
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await insertObject(db, 'report', {
+    id: generateId(),
+    code: SCHOOL_REPORT_CODE,
+    config: {
+      fetch: {
+        dataElements: [
+          'SchPop009',
+          'SchPop010',
+          'SchPop011',
+          'SchPop012',
+          'SchPop013',
+          'SchPop014',
+          'SchPop015',
+          'SchPop016',
+          'SchPop017',
+          'SchPop018',
+          'SchPop019',
+          'SchPop020',
+          'SchPop021',
+          'SchPop022',
+          'SchPop023',
+          'SchPop024',
+          'SchPop025',
+          'SchPop026',
+          'SchPop027',
+          'SchPop028',
+          'SchPop029',
+          'SchPop030',
+          'SchPop031',
+          'SchPop032',
+          'SchPop033',
+          'SchPop034',
+          'SchDISmr',
+          'SchCVD026b',
+          'SchFDpho',
+        ],
+      },
+      transform: [
+        'keyValueByDataElementName',
+        'mostRecentValuePerOrgUnit',
+        {
+          transform: 'select',
+          '...': ['organisationUnit'],
```

Comment:
Remind me what this does?

Reply:
This just includes the passed in org unit in the results of the report, probably not entirely necessary for our usage

Reply:
Aha :+1: